### ISSUE TITLE
Crash on upload picture with right to left languages

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#1163]
Change your phone language to Arabic, try to upload a picture. It results this crash:
`02-16 00:32:56.294 32006-32006/fr.free.nrw.commons.debug E/AndroidRuntime: FATAL EXCEPTION: main
                                                                           Process: fr.free.nrw.commons.debug, PID: 32006
                                                                           java.lang.NullPointerException
                                                                               at fr.free.nrw.commons.upload.SingleUploadFragment.titleInfo(SingleUploadFragment.java:232)
                                                                               at fr.free.nrw.commons.upload.SingleUploadFragment_ViewBinding$1.onTouch(SingleUploadFragment_ViewBinding.java:46)
                                                                               at android.view.View.dispatchTouchEvent(View.java:7772)`

**Comment by diddypod** [comment]
@neslihanturan I was unable to replicate the error and uploaded a picture successfully with the phone language to Arabic. Could you share some more details?

**Comment by neslihanturan** [comment]
Sure, test device API 4.4.2, whenever I click to edit title or description it gives me:
![npe](https://user-images.githubusercontent.com/3127881/36339965-aa35fd3e-13e1-11e8-8bc9-b3ab8d976b4f.png)
NPE on line 232 of SingleUploadFragment.java
```

        //Should replace right with end to support different right-to-left languages as well
        final int value = titleEdit.getRight() - titleEdit.getCompoundDrawables()[2].getBounds().width();
```

**Comment by diddypod** [comment]
Successfully replicated the error.
Starting work on it :)

**Comment by maskaravivek** [comment]
Awesome @diddypod. Let us know if you need any help with it. :)

**Comment by diddypod** [comment]
I've fixed it and created a PR ([#1172](https://github.com/commons-app/apps-android-commons/pull/1172)). Please check :)
